node/tests/RedisClusterClient.test.ts

```diff
@@ -35,12 +35,31 @@ const TIMEOUT = 10000;
 describe("RedisClusterClient", () => {
     let testsFailed = 0;
     let cluster: RedisCluster;
+    let client: RedisClusterClient;
     beforeAll(async () => {
-        cluster = await RedisCluster.createCluster(true, 3, 0);
+        const args = process.argv.slice(2);
```

Comment:
Can you reuse this piece of code by creating a function?

Reply:
why not using parseArg https://simonplend.com/command-line-argument-parsing-with-node-js-core/